docs: miscellaneous updates (edge) (#1936)
spectrocloud/librarium: docs/docs-content/clusters/edge/networking/vxlan-overlay.md

    </TabItem>
    </Tabs>

9. Review the rest of your cluster profile values and make changes as needed. Click on **Next**.

10. In the **Cluster Config** stage, toggle on **Enable Overlay Network**. This will prompt you to provide additional configuration for your virtual overlay network. 

> 11. In the **Overlay CIDR Range** field, provide a private IP range for your cluster to use. Ensure that this range is not used by others in the same network environment. When you toggle on **Enable Overlay Network**, Palette provides with a default commonly unused range. We suggest you keep the default range unless you have a specific IP range you want to use. 

Reviewer: ```suggestion
11. In the **Overlay CIDR Range** field, provide a private IP range for your cluster to use. Ensure this range is not used by others in the same network environment. When you toggle on **Enable Overlay Network**, Palette provides a default range that is commonly unused. We suggest you keep the default range unless you have a specific IP range you want to use. 
```

3. Select the host cluster you created to view its details page.

> 4. Select the **Nodes** tab, in the **Private Ips** column, each host has an overlay IP address within the CIDR range you provided during cluster configuration. 

Reviewer: This is a run-on sentence. Is the following intended?

Select the **Nodes** tab. For each host, the **Private Ips** column lists an overlay IP address within the CIDR range you provided during cluster configuration.

> 4. Select the **Nodes** tab, in the **Private Ips** column, each host has an overlay IP address within the CIDR range you provided during cluster configuration. 

Author: ```suggestion
4. Select the **Nodes** tab. For each host, the **Private Ips** column lists an overlay IP address within the CIDR range you provided during cluster configuration.
```
